ietf-network-inventory.yang

```diff
@@ -65,18 +65,11 @@ module ietf-network-inventory {
      described in BCP 14 (RFC 2119) (RFC 8174) when, and only when,
      they appear in all capitals, as shown here.";
 
-  // RFC Ed.: replace XXXX with actual RFC number and remove this
-  // note.
-  // RFC Ed.: update the date below with the date of RFC publication
-  // and remove this note.
-
   revision 2024-04-09 {
     description
       "Initial version";
     reference
       "RFC XXXX: A YANG Data Model for Network Inventory.";
-    //RFC Editor: replace XXXX with actual RFC number, update date
-    //information and remove this note
```

Comment:
See my comment in #54: https://github.com/ietf-ivy-wg/network-inventory-yang/pull/54#discussion_r1781561314

Reply:
2024-10-01

The note to keep is the one above:

```
  // RFC Ed.: update the date below with the date of RFC publication
  // and remove this note.

```